e2e/testcafe-devextreme/tests/dataGrid/editing/editingEvents.ts

```diff
@@ -172,3 +174,52 @@ testCases.forEach(({ caseName, expected, onRowRemoving }) => {
     });
   });
 });
+
+// T1250405
+test('DataGrid - Canceled rows are hidden when multiple rows are added in batch mode', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const addBtn = dataGrid.getToolbar().getItem();
+  const saveBtn = dataGrid.getToolbar().getItem(1);
+  await t
```

Comment:
Please add a check that the grid is ready before starting all actions - `await t.expect(dataGrid.isReady()).ok()`.
This will increase the stability of the test.